app/helpers/findings_helper.rb

```diff
@@ -413,4 +413,14 @@ def summary_for review
 
       "#{ConclusionReview.human_attribute_name 'summary'}: #{summary}"
     end
+
+    def finding_impact_risks_types finding
+      types = {}
+
+      finding.amount_by_impact.reverse_each do |id, name|
+        types.merge!(name => id)
+      end
+
+      types.to_a.to_json
+    end
```

Comment:
No le había prestado bien atención a la lógica acá, pero estás creando un hash desde un hash solo por invertirlo. Eso se hace con invert, sin probarlo mucho esto debería ser:
```suggestion
      finding.amount_by_impact.invert.to_a.to_json
    end
```

Reply:
hace el invert y a su vez le cambia el orden

Reply:
finding.amount_by_impact.invert.reverse_each.to_json lo cambie por esto

Reply:
Ok, joya.